modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -466,7 +466,7 @@ VALUES ('details_providing_medicine', '9911', 'Codes', NULL);
 
 #IfNotRow manage_templates_letters letter_name letter_x_ray
 INSERT INTO `manage_templates_letters` (`id`, `letter_name`, `letter_class`, `letter_class_action`, `active`, `letter_post_json`) VALUES
-(1, 'letter_x_ray', 'EmergencyMedicine\\Controller\\xrayLetterController', 'pdf', 1, '\n{"facility": "required","encounter": "required","owner": "optional","patient": "optional"}');
+(1, 'letter_x_ray', 'EmergencyMedicine\\Controller\\XrayLetterController', 'pdf', 1, '\n{"facility": "required","encounter": "required","owner": "optional","patient": "optional"}');
```

Comment:
write correct condition if you change it.
It's alreay in used in the develop

Reply:
will this work ? 

#IfNotRow2D manage_templates_letters letter_name letter_x_ray letter_class EmergencyMedicine\\Controller\\XrayLetterController

Reply:
need to test